R/GeostTextureLibrarySharedRoxy.R

```diff
@@ -396,7 +395,8 @@ CalcMeans=function(deltas,w,exponent){
 #' @param kernels The kernels to be used for computing the directional differences (e.g. order 1 or 2 for various lags)
 #' @param w The moving window adopted for computing the geostatistical index (i.e., MAD)
 #' @param exponent The exponent: increasing the exponent increase the sensitivity to outliers. Set 2 for Variogram and 1 for Madogram.
-#' @return A list of 3 rasters: 1)isotropic roughness; 2) direction of anisotropy; 3)index of anisotropy.
+#' @return A SpatRaster with 3 layers: 1)isotropic roughness; 2) direction of anisotropy; 3)index of anisotropy.
```

Comment:
yes, you are right it is a 3 layers raster not a list!

Reply:
Cool, I saw "list of rasters" mentioned a few times on the input and output side of the documentation so it's probably worth a quick check to change those over to saying "multilayer SpatRaster" in the documentation.